src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -698,8 +698,8 @@ export async function getFeatureIdType(params: CommonParams, userFeatureId: stri
 
 	// Legacy feature-set ID
 	if (!userFeatureId.includes('/') && !userFeatureId.includes('\\')) {
-		output.write(`Legacy feature '${userFeatureId}' not supported. Please check https://containers.dev/features for replacements.`, LogLevel.Error);
-		throw new ContainerError({ description: `Legacy feature '${userFeatureId}' not supported. Please check https://containers.dev/features for replacements.` });
+		output.write(`Legacy feature '${userFeatureId}' forgot to prepend './' at the begining of the feature. Please check https://containers.dev/implementors/features-distribution/#addendum-locally-referenced for replacements.`, LogLevel.Error);
```

Comment:
Can you follow the messaging in https://github.com/devcontainers/cli/issues/834#issuecomment-2148074401? Thanks! 

We need to append the new message and not replace it. The issue comment has the needed message string.